data/tcr_epitope_binding/meta.yaml

```diff
@@ -0,0 +1,44 @@
+name: tcr_epitope_binding
+description: |-
+  T-cells are an integral part of the adaptive immune system, whose survival, proliferation, activation
+          and function are all governed by the interaction of their T-cell receptor (TCR) with immunogenic peptides (epitopes).
+          A large repertoire of T-cell receptors with different specificity is needed to provide protection against a wide range of pathogens.
+          This new task aims to predict the binding affinity given a pair of TCR sequence and epitope sequence.
+targets:
+- id: binding
+  description: TCR epitope binding.
+  units: ''
+  type: binary classification
+  names:
+  - tcr binding affinity
+  - binding affinity
+  - binding
```

Comment:
I feel it would be better if we could include in all the "synonyms" also the binding site, e.g., "epitope binding affinity"

Reply:
@kjappelbaum I included "epitope binding affinity" and also "epitope binding" as synonyms.